google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableExporterUtils.java

```diff
@@ -104,9 +104,10 @@ static List<TimeSeries> convertCollectionToListOfTimeSeries(
     List<TimeSeries> allTimeSeries = new ArrayList<>();
 
     for (MetricData metricData : collection) {
-      // TODO: scope will be defined in BuiltinMetricsConstants. Update this field in the following
-      // PR.
-      if (!metricData.getInstrumentationScopeInfo().getName().equals("bigtable.googleapis.com")) {
+      if (!metricData
```

Comment:
please add a comment, something along the lines of "Filter out metric data for instruments that are not part of the bigtable builtin metrics meter